getting_started/scripting/gdscript/gdscript_styleguide.rst

```diff
@@ -86,6 +86,13 @@ Surround functions and class definitions with two blank lines:
 
 Use one blank line inside functions to separate logical sections.
 
+Line length
+~~~~~~~~~~~
+
+Try to keep lines under 80 characters. This ensures greater readability on small
```

Comment:
how about `at most 79 characters`? Meaning is the same, but the threshold (79) is explicit  - one does not have to think about it.

Reply:
This may sound weird, but I've found 80 characters to be significantly less limiting compared to 79 characters…

I can change the wording to make it more explicit though.

Reply:
okay then :)